samples/cpp/text2image/main.cpp

```diff
@@ -44,10 +15,10 @@ int32_t main(int32_t argc, char* argv[]) try {
     ov::Tensor image = pipe.generate(prompt,
         ov::genai::width(512),
         ov::genai::height(512),
-        ov::genai::num_inference_steps(20),
-        ov::genai::num_images_per_prompt(1));
+        ov::genai::num_inference_steps(20));
```

Comment:
Shouldn't we show the usage of `num_images_per_prompt` param there?

Reply:
reverted